etc/scripts/json2csv.py

```diff
@@ -246,7 +244,7 @@ def cli(json_input, csv_output):
 
     JSON_INPUT is either a ScanCode json format scan or the data.json file from a ScanCode html-app format scan.
 
-    Paths will be prefixed with '/code/' to provide a common base directory for scanned resources.
+    #Paths will be prefixed with '/code/' to provide a common base directory for scanned resources.
```

Comment:
@JonoYang  so make function named --prepend-root  and with some condition.

Then which condition should I put? Please help